examples/rust_reverse/test/test_rust_reverse.rb

```diff
@@ -10,4 +10,10 @@ def test_that_it_has_a_version_number
   def test_it_does_something_useful
     assert_equal "dlrow olleh", RustReverse.reverse("hello world")
   end
+
+  def test_stressing_it_out
+    10000.times do
+      assert_equal "a" * 100000, RustReverse.reverse("a" * 100000)
+    end
+  end
```

Comment:
Another thing we lose from not using ruby's allocation functions is ability to use [`GC.stress=`](https://ruby-doc.org/core-3.1.2/GC.html#method-c-stress-3D) to force all the allocations to invoke the garbage collector for testing.  This can make it easier for ruby extension gems to be tested for properly marking values or using `RB_GC_GUARD(obj)` to make the code paths tested with GC stressed aren't going to access GC freed memory in production.